core/src/main/java/com/cloud/agent/api/StopCommand.java

```diff
@@ -36,6 +36,7 @@
     String controlIp = null;
     boolean forceStop = false;
     private Map<String, DpdkTO> dpdkInterfaceMapping;
+    Map<String, Boolean> vlanToPersistenceMap;
```

Comment:
same as before, NPE check for consumer

Reply:
There's an explicit check done at the consuming end to ensure this isn't null